docs/topics/signals.rst

```diff
@@ -118,7 +118,14 @@ engine_started
 
 .. note:: This signal may be fired *after* the :signal:`spider_opened` signal,
     depending on how the spider was started. So **don't** rely on this signal
-    getting fired before :signal:`spider_opened`.
+    getting fired before :signal:`spider_opened`.   
+
+    If you specifically need a fully initialized :param Crawler: instance within the
```

Comment:
`:param Crawler:` looks wrong, "specifically" looks out of place here and overall I don't think it's recommended over `start_requests`.

Reply:
Oh and it's definitely not helpful with getting a fully initialized crawler inside `Spider.from_crawler()`.